src/scrabBag.js

```diff
@@ -0,0 +1,63 @@
+export default function scrabBag(letters){
+  let bag = new Scrab
+  bag.remove(letters)
+  return(bag.output())
+}
+
+class Scrab{
```

Comment:
So after talking with an SEP you could declare this as a const and then it would make it easier for the functions you write to interact with this object a little more concise